Confirmation is now optional. (#270)
datatrails/datatrails-python: README.rst

            # it does not start with arc_
        }
        #
        # There are 3 alternatives
        #
        # 1. Create the asset:
        #    The first argument is the attributes of the asset
        return arch.assets.create(attrs=attrs)
        #
        # 2. alternatively one can wait for the asset to be confirmed in the
        #    immutable store.
        #    The second argument is wait for confirmation:
        #      If @confirm@ is True then this function will not
        #      return until the asset is confirmed.
        #
        # Confirmation guarantees that 3rd parties can retrieve and cryptographically
        # verify your Assets, which can take a few seconds to propagate. It is typically
        # not necessary to wait unless your workflow involves near-real-time

>         # communication with 3rd parties and the 3rd party needs instant cryptographic

Reviewer: ```suggestion
        # communication with 3rd parties and the 3rd party needs immediate cryptographic
```

Author: fixed

the yaml representation of the request body for an asset or event.

The confirm: field is a control variable for the PythonSDK that ensures that the
asset or event is confirmed before returning. This is optional and is only required

> if guarantees that 3rd parties can retrieve and cryptographically

Reviewer: ```suggestion
if 3rd parties need to immediately retrieve and cryptographically
```

Author: fixed
